pkg/scheduler/framework/plugins/interpodaffinity/filtering.go

```diff
@@ -337,22 +254,20 @@ func (pl *InterPodAffinity) PreFilter(ctx context.Context, cycleState *framework
 		return framework.NewStatus(framework.Error, fmt.Sprintf("failed to list NodeInfos with pods with affinity: %v", err))
 	}
 
+	podInfo := framework.NewPodInfo(pod)
```

Comment:
For a follow up: Perhaps we could store something in QueuedPodInfo? This way processed terms are stored across scheduling attempts. And this is something we could also add to Spreading plugin.